modules/nf-core/bcftools/pluginimputeinfo/meta.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: "bcftools_pluginimputeinfo"
+description: Adds imputation information metrics to the INFO field based on selected FORMAT tags. Only the IMPUTE2 INFO metric from FORMAT/GP tags is currently available.
+keywords:
+  - impute-info
+  - bcftools
+  - VCF
+tools:
+  - "bcftools":
+      description: "BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations."
+      homepage: "http://samtools.github.io/bcftools/bcftools.html"
+      documentation: "https://samtools.github.io/bcftools/howtos/plugins.html"
+      tool_dev_url: "https://github.com/samtools/bcftools"
+      doi: "10.1093/bioinformatics/btp352"
+      licence: ["MIT"]
```

Comment:
Missing args reference https://nf-co.re/docs/guidelines/components/modules#documentation-of-args-of-each-piped-or-multiple-command